portal_sale_hide_download/views/sale_portal_templates.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="sale_order_portal_template"
+        name="Sales Order Portal Template"
+        inherit_id="sale.sale_order_portal_template"
+    >
+        <xpath expr="//div[hasclass('o_download_pdf')]" position="attributes">
+            <attribute name="t-att-style">'display: none;'</attribute>
```

Comment:
```suggestion
            <attribute name="style">display: none;</attribute>
```

I'm not perfectly certain but my understanding is that we do not need to use `t-att-style` if the content is static.